tests/test_syncing.py

```diff
@@ -73,3 +88,18 @@ def test_sync_device_during_onboarding(multiple_instance, user_data):
                 lambda: user_canvas.is_user_image_contains(user.name[:2]),
                 configs.timeouts.UI_LOAD_TIMEOUT_MSEC
             )
+
+
+@allure.testcase('https://ethstatus.testrail.net/index.php?/cases/view/703631', 'Wrong sync code')
+@pytest.mark.case(703631)
+@pytest.mark.parametrize('wrong_sync_code', [
+    pytest.param('cs3:scrCo2f:Jni:mMDryn4AMHBoFTQfp9mPeVMaz4FiYfaCfpRokUvvJNsM:BR5WPtAZLRXj1Xw21azbiVG25jSfAfvZH8nLo2o')
```

Comment:
can we indeed use a string that is not a sync code? The one above looks as sync code :)

Reply:
Actually I did it on purpose. Sync code with a wrong symbol In it. But of course I can add something different

Reply:
Maybe I can add more to check - different sync codes (sync code with mistake in it, something which is not a sync code at all like one symbol or smth, maybe sth else)?

Reply:
i talked to Magnus about it - lets just replace the value with a random string and merge. We add more scenarios later, or expand this one some day

Reply:
Ok. Will do this